testng-eclipse-plugin/META-INF/MANIFEST.MF

```diff
@@ -18,7 +18,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.jdt.core,
  org.eclipse.jdt.debug.ui,
  org.eclipse.jdt.junit,
- org.eclipse.jdt.launching,
+ org.eclipse.jdt.launching;bundle-version="3.10",

```

Comment:
the new methods/constants used for supporting Java9+ in TestNGLaunchConfigurationDelegate.java requires JDT 3.10 (aka, Eclipse photon)